### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EditMetada_ScreenReader: Narrator does not read the 'Code' written in the code block Properly.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4013]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10

**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Click on Run or right click and select Run to run the notebook.

4.Navigate to any file present under Files tab-->Hit enter-->View Menu-->Cell Toolbar-->Edit Metadata. Click on Edit Metadata button appeared on page.
5.Start Narrator and Verify how Narrator reads the code present in the code block.

**Actual**
Narrator does not read the code written inside the code block properly.

**Note
This issue exist for 'Chrome + NVDA' configuration too.

Similar issue exist for below page**
CreationAndDeployment Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.

**Expected**
Narrator should read the code written inside the code block clearly.

**User Impact**
Screenreader user will not be able to understand what code is written inside the code block.
**Attachment:**

[A11y_AzureNotebooks_Jupyter_EditMetada_ScreenReader20.pptx](https://github.com/jupyter/notebook/files/2401286/A11y_AzureNotebooks_Jupyter_EditMetada_ScreenReader20.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
